Sources/Shared/Models/VIMVideo.m

```diff
@@ -160,6 +161,11 @@ - (Class)getClassForObjectKey:(NSString *)key
         return [FileTransfer class];
     }
 
+    if ([key isEqualToString:@"publish_to_social_job"])
```

Comment:
Should be `publish_to_social`
https://github.vimeows.com/Vimeo-API/design/pull/148/files#diff-4dafa2f9b34cb46c64cb8ef8d9694149R124